### PR TITLE
Kieran's first pull request

### PR DESCRIPTION
**Pull request by KGibb8**
There's plenty more work to do on this. Going to keep cracking on at it for fun and momentum but for the time being I wanted to submit what I have done.

### CODE REVIEW

File: primes.rb [hunk 1]
```diff
@@ -1,6 +1,41 @@
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def even?(n)
+    if n % 2 == 0
+      even = true
+    else
+      even = false
+    end
```
Comment (dmgarland) [inline primes.rb]: This is cool, but note that Ruby has this out of the box e.g. `n.even?`

File: views/fieldinput.erb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<form action="/update" method="POST">
+  <fieldset>
+    <table>
+      <tbody>
+        <tr>
+          <td>
+            <div class="profile-section box">
+                <label for="trainee[about]">About</label>
+                <br>
+                <textarea rows="3" cols="40" name="trainee[about]" value="<%= @personal_info["about"] %>"></textarea>
```
Comment (dmgarland) [inline views/fieldinput.erb]: With textarea, the value goes in between the textarea tags as a text node. Try it and it should show up

File: views/fieldinput.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            </div>
+          </td>
+        </tr>
+        <tr>
+          <% count = 1 %>
+          <% @profile.each do |field, value| %>
+          <td>
+            <div class="profile-section box">
+              <label for="trainee[<%= field %>]"><%= field.capitalize.tr("_"," ")%></label>
+              <input type="text" name="trainee[<%= field %>]" value="<%= @profile[ value ] %>"/>
```
Comment (dmgarland) [inline views/fieldinput.erb]: Perhaps value itself might be better here?

File: views/fieldinput.erb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            </div>
+          </td>
+          <% if count.even? %>
+        </tr>
+        <tr>
+          <% count += 1 %>
+          <% else %>
+          <% count += 1 %>
+          <% end %>
+          <% end %>
```
Comment (dmgarland) [inline views/fieldinput.erb]: Watch your indentation here. It helps to spot bugs early on

**Comment by dmgarland** [comment]
Well done, not 100% sure about the green background but looking good ;)

File: views/fieldinput.erb (hunk 2 above)
Comment (KGibb8) [inline views/fieldinput.erb]: Thanks worked a treat!

**Pull request closed without merging**